processing/processing/main.py

```diff
@@ -92,11 +92,18 @@ def get_preprocessing_result(
         pages,
         current_tenant,
     )
-    bucket_name = convert_bucket_name_if_s3prefix(current_tenant)
-    return Response(
-        content=send_preprocess_result(bucket_name, file_id, pages),
-        media_type="application/json",
-    )
+    try:
+        bucket_name = convert_bucket_name_if_s3prefix(current_tenant)
+        return Response(
+            content=send_preprocess_result(bucket_name, file_id, pages),
+            media_type="application/json",
+        )
+    except Exception:
+        logger.exception("Cannot fin preprocessing results")
+        return Response(
```

Comment:
This would work, but you can also use `JsonResponse(content=[])` to make it cleaner

Reply:
Just want to use existing code style. To change `Response` to `JsonResponse` will lead us to change `send_preprocess_result` response. 
In case we will add `JsonResponse` with `Response`, we need to change `get_preprocessing_result` method signature to return `Union[Response, JsonResponse]`. Seems that too much effort for that. 
However, it could be done when we will unify BadgerDoc responses

Reply:
Yeah, it is not mandatory right now; JsonResponse inherit from Response and just setup media_type to json, that's it